src/app/games/new-game/NewGameDummyMinimax.ts

```diff
@@ -0,0 +1,48 @@
+import { Minimax } from 'src/app/jscaip/Minimax';
+import { NewGameMove } from './NewGameMove';
+import { NewGameState } from './NewGameState';
+import { NewGameBoardValue, NewGameLegalityInfo, NewGameNode, NewGameRules } from './NewGameRules';
+
+/**
+ * What is called "minimaxes" here are heuristic calculators.
+ * They assign a score to a state so that the `LocalGameWrappercomponent` can
+ * use it within the AI system to know which move to play.
+ *
+ * By convention, we call such a class a "dummy minimax" if:
+ *   - it doesn't filter any move in `getListMoves`
+ *   - it has a simplistic `getBoardValue`, only relying the score stored in the state
+ */
+export class NewGameDummyMinimax extends Minimax<NewGameMove,
+                                                 NewGameState,
+                                                 NewGameLegalityInfo,
+                                                 NewGameBoardValue,
+                                                 NewGameRules>
+{
+    /**
+     * This method lists all useful moves to consider in the move exploration.
+     * In general, you can list all possible moves.
+     * It is however an option to exclude some move of the list if:
+     *     - they have no way to be needed in any interesting part
+     *     - the can only lead to defeat
```

Comment:
to unevitable defeat, cause if you cannot avoid it, the function cannot block the game :p